modules/homebrew/init.zsh

```diff
@@ -21,7 +21,20 @@ fi
 # Load 'HOMEBREW_' prefixed variables only. Avoid loading 'PATH' related
 # variables as they are already handled in standard zsh configuration.
 if (( $+commands[brew] )); then
-  eval "${(@M)${(f)"$(brew shellenv 2> /dev/null)"}:#export HOMEBREW*}"
+  cache_file="${XDG_CACHE_HOME:-$HOME/.cache}/prezto/brew-shellenv-cache.zsh"
+  if [[ "${commands[brew]}" -nt "$cache_file" \
+      || "${ZDOTDIR:-$HOME}/.zpreztorc" -nt "$cache_file" \
+      || ( ! -n ${cache_file}(#qNmw-1) ) ]]; then
```

Comment:
This will require `EXTENDED_GLOB` to be set. This might not be the case when one enables this module. We have burned ourselves in the [past](https://github.com/sorin-ionescu/prezto/issues/1949#issuecomment-902188772) :)

Do you think we can ignore checking for the age of the `cache_file`?

Reply:
Thanks for pointing this out. I can either do `LOCAL_OPTIONS` + `EXTENDED_GLOB`, or I can remove the check for the age. I don't really need the age myself, but I think of it as some kind of an escape hatch - just in case we missed some dependency and brew's shellenv does get updated, we'll eventually pick it up. What do you think?